apps/andi/assets/css/variables.scss

```diff
@@ -1,5 +1,6 @@
 $lightgray: lightgray;
 $darkgray: darkgray;
+$fadedblack: rgba(0, 0, 0, 0.6);
```

Comment:
Is there a non-color name for this color?  Something like `header-shadow`?  I'd like to do the same with the grays/greys, but I know you didn't add those.

Reply:
We can fix that